website/content/docs/services/services.mdx

```diff
@@ -0,0 +1,39 @@
+---
+layout: docs
+page_title: Services Overview
+description: >-
+  Learn about services and service discovery workflows and concepts for virtual machine environments. 
+---
+
+# Services Overview
+This topic provides overview information about services and how to make them discoverable Consul when your network operates on virtual machines. If service mesh is enabled in your network, refer to the topics in [Consul Service Mesh](/consul/docs/connect/). If your network is running in Kubernetes, refer to the topics in [Consul on Kubernetes](/consul/docs/k8s). 
```

Comment:
Dealer's choice! I put "with".. could be "through".. or "in".. or something else.
```suggestion
This topic provides overview information about services and how to make them discoverable with Consul when your network operates on virtual machines. If service mesh is enabled in your network, refer to the topics in [Consul Service Mesh](/consul/docs/connect/). If your network is running in Kubernetes, refer to the topics in [Consul on Kubernetes](/consul/docs/k8s). 
```